OBF Bylaws.md

```diff
@@ -95,16 +94,12 @@ ARTICLE IV - NOMINATIONS AND ELECTIONS
     a.	Nominees or appointees for Director positions (Board Members at Large) must be voting members of the OBF in good standing.
     b.	Nominees or appointees for Officer positions must have been voting members of the OBF for a minimum of one year, be in good standing, and have previously served on the Board of Directors for a minimum of one year.
     c.	The Board of Directors may waive qualification requirements at its discretion by unanimous consent of attending Directors, provided the quorum is met. If the membership requirement is waived for a nominee, the nominee automatically becomes a member upon election, and must meet all requirements to be in good standing within 4 (four) weeks after election. 
-7.	Nominating Committee
-    a.	The committee shall consist of the Parliamentarian and a minimum of 2 (two) active members of the OBF.
-    b.	The committee shall attempt to contact members of the Board of Directors and active members of the OBF in an effort to identify eligible members interested in serving on the Board of Directors, and to solicit nominations from the general membership.
-    c.	The committee shall present the names of candidates to the general membership prior to the annual Board meeting.
-    d.	The committee shall prepare an electronic or paper ballot of candidates to be used at the annual meeting for the purpose of voting.
+7.	_Removed_.
 8.	Paper ballots shall be counted by 2 (two) active members of the OBF selected by the President. Electronic ballots will be counted electronically.
 9.	Election results shall be recorded and filed in the minutes.
 
 ARTICLE V - BYLAW AMENDMENTS
 ---------------------------
 
-1. These bylaws shall be amended by a 2/3 (two-thirds) majority vote of the Board of Directors at any meeting of the Board, provided the quorum is met and that prior written or email notice has been given to the membership (as in Article II, Item 4).
-2. These bylaws shall be reviewed every 2 (two) years by the committee convened and chaired by the Parliamentarian and 2 (two) active members of the OBF nominated by the Parliamentarian.
+1. These bylaws shall be amended by a 2/3 (two-thirds) majority vote of the Board of Directors at any public meeting of the Board, provided the quorum is met and that prior written or email notice has been given to the membership (as in Article II, Item 4).
+2. These bylaws shall be reviewed every 2 (two) years by a special committee convened by the Board of Directors.
```

Comment:
Should we say the committee might just be the full board? Or is this sufficiently flexible as written?